docker/README.md

```diff
@@ -1,91 +1,145 @@
+### 2C. MongoDB runs on a different host
 
-As a result of this command, there is a context broker listening on port 1026 on localhost. Try to see if it works now with
+If you want to connect to a different MongoDB instance do the following command **instead of** the previous one
+
+	sudo docker run -d --name orion1 -p 1026:1026 fiware/orion -dbhost <MongoDB Host>
+
+Check that everything works with
 
 	curl localhost:1026/version
+----
+## 3. Build a docker image
+
+Building an image gives more control on what is happening within the Orion Context Broker container. Only use this method if you are familiar with building docker images or really need to change how this image is built. For most purposes you probably don't need to build an image, only deploy a container based on one already built for you (which is covered in sections 1 and 2).
 
+Steps:
 
+1. Download [Orion's source code](https://github.com/telefonicaid/fiware-orion/) from Github (`git clone https://github.com/telefonicaid/fiware-orion/`)
+2. `cd fiware-orion/docker`
+3. Modify the Dockerfile to your liking
+4. Run Orion...
+	* Using an automated scenario with docker-compose and building your new image: `sudo docker-compose up`. You may also modify the provided `docker-compose.yml` file if you need so.
+	* Manually, running MongoDB on another container: 
+		1. `sudo docker run --name mongodb -d mongo:2.6`
+		2. `sudo docker build -t orion .`
+		3. `sudo docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 orion -dbhost mongodb`.
+	* Manually, specifying where to find your MongoDB host:
+		1. `sudo docker build -t orion .`
+		2. `sudo docker run -d --name orion1 -p 1026:1026 orion -dbhost <MongoDB Host>`.
+
+Check that everything works with
+
+	curl localhost:1026/version
 
-A few points to consider:
+The parameter `-t orion` in the `docker build` command gives the image a name. This name could be anything, or even include an organization like `-t org/fiware-orion`. This name is later used to run the container based on the image.
 
-* The name `orion1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
-* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system,
-whose name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
-* In `-p 1026:1026` the first value represents the port to listen on localhost. If you want to run a second context broker
+If you want to know more about images and the building process you can find it in [Docker's documentation](https://docs.docker.com/userguide/dockerimages/).
+
+## 4. Other info
+
+Things to keep in mind while working with docker containers and Orion Context Broker.
+
+### 4.1 Data persistence
+Everything you do with Orion Context Broker when dockerized is non-persistent. *You will lose all your data* if you turn off the MongoDB container. This happens with either this method or the manual one.
```

Comment:
Good catch. It was actually a left over from the previous version.

Fixed in 830e21d